UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Services/FileShareService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.Extensions.Options;
+using System;
+using System.IO;
+using UKHO.SalesCatalogueFileShareServicesMock.API.Common;
+using UKHO.SalesCatalogueFileShareServicesMock.API.Helpers;
+using UKHO.SalesCatalogueFileShareServicesMock.API.Models.Response;
+
+namespace UKHO.SalesCatalogueFileShareServicesMock.API.Services
+{
+    public class FileShareService
+    {
+        private readonly IOptions<FileShareServiceConfiguration> fileShareServiceConfiguration;
+
+        public FileShareService(IOptions<FileShareServiceConfiguration> fileShareServiceConfiguration)
+        {
+            this.fileShareServiceConfiguration = fileShareServiceConfiguration;
+        }
+
+        public SearchBatchResponse GetBatches(string filter)
+        {
+            if (!string.IsNullOrEmpty(filter))
+            {
+                return FileHelper.ReadJsonFile<SearchBatchResponse>(fileShareServiceConfiguration.Value.FileDirectoryPath + fileShareServiceConfiguration.Value.ScsResponseFile); 
+            }
+            return null;
+        }
+
+        public byte[] GetENCFilePath(string filesName)
```

Comment:
`GetENCFilePath()` will always return the _first_ TXT or 000 file in the directory - only the extension of the `filesName` parameter is used by the file selection logic.

Please ignore this comment if this is the intended behaviour

Reply:
Thanks for highlighting it, making the changes accordingly.